packages/world/ts/objects.ts

```diff
@@ -0,0 +1,520 @@
+  ],
+  Plank: [
+    { name: "AnyPlank", mass: 4500000000000000n },
+    { name: "OakPlanks", mass: 4500000000000000n },
+    { name: "BirchPlanks", mass: 4500000000000000n },
+    { name: "JunglePlanks", mass: 4500000000000000n },
+    { name: "SakuraPlanks", mass: 4500000000000000n },
+    { name: "SprucePlanks", mass: 4500000000000000n },
+    { name: "AcaciaPlanks", mass: 4500000000000000n },
+    { name: "DarkOakPlanks", mass: 4500000000000000n },
+    { name: "MangrovePlanks", mass: 4500000000000000n },
+  ],
+  OreBlock: [
+    { name: "CopperBlock", mass: 675000000000000000n },
+    { name: "IronBlock", mass: 675000000000000000n },
+    { name: "GoldBlock", mass: 14400000000000000000n },
+    { name: "DiamondBlock", mass: 45000000000000000000n },
+    { name: "NeptuniumBlock", mass: 45000000000000000000n },
+  ],
+  Seed: [
+    {
+      name: "WheatSeed",
+      energy: 10000000000000000n,
+      timeToGrow: 900n,
+      crop: "Wheat",
+    },
+    {
+      name: "PumpkinSeed",
+      energy: 10000000000000000n,
+      timeToGrow: 3600n,
+      crop: "Pumpkin",
+    },
+    {
+      name: "MelonSeed",
+      energy: 10000000000000000n,
+      timeToGrow: 3600n,
+      crop: "Melon",
+    },
+  ],
+  Sapling: [
+    { name: "OakSapling", energy: 148000000000000000n, timeToGrow: 345600n },
+    { name: "BirchSapling", energy: 139000000000000000n, timeToGrow: 345600n },
+    { name: "JungleSapling", energy: 300000000000000000n, timeToGrow: 345600n },
+    { name: "SakuraSapling", energy: 187000000000000000n, timeToGrow: 345600n },
+    { name: "AcaciaSapling", energy: 158000000000000000n, timeToGrow: 345600n },
+    { name: "SpruceSapling", energy: 256000000000000000n, timeToGrow: 345600n },
+    {
+      name: "DarkOakSapling",
+      energy: 202000000000000000n,
+      timeToGrow: 345600n,
+    },
+    {
+      name: "MangroveSapling",
+      energy: 232000000000000000n,
+      timeToGrow: 345600n,
+    },
+  ],
+  Station: [
+    { name: "Furnace", mass: 108000000000000000n },
+    { name: "Workbench", mass: 17800000000000000n },
+    { name: "Powerstone", mass: 3735000000000000000n },
+  ],
+  SmartEntityBlock: [
+    { name: "ForceField", mass: 3735000000000000000n, isMachine: true },
+    { name: "Chest", mass: 35600000000000000n },
+    { name: "SpawnTile", mass: 9135000000000000000n },
+    { name: "Bed", mass: 13350000000000000n },
+  ],
+  MiscBlock: [
+    // TODO: must assign some of these to terrain categories
+    { name: "RedMushroomBlock", mass: 12500000000000000n },
+    { name: "BrownMushroomBlock", mass: 12500000000000000n },
+    { name: "MushroomStem", mass: 12500000000000000n },
+    { name: "Cactus", mass: 1300000000000000n },
+    { name: "Snow", mass: 4000000000000000n },
+    { name: "Ice", mass: 4000000000000000n },
+    { name: "SpiderWeb", mass: 100000000000000n },
+    { name: "Bone", mass: 1000000000000000n },
+    { name: "TextSign", mass: 17800000000000000n },
```

Comment:
TextSign should be in SmartEntityBlock

Reply:
is it? I thought we were not going to use programs for signs

Reply:
ahh no we're not going to support content of signs in the physics, which means all content has to come from programs right